assets/chat/css/style.scss

```diff
@@ -1491,6 +1502,23 @@ hr {
       &.creator {
         color: $text-color1;
       }
+      &.favorite {
+        cursor: pointer;
+        margin-top: 5px;
+        .btn-icon {
+          transition: background-color 500ms;
+          width: 20px;
+          height: 20px;
+          mask: url('../img/icon-favorite.svg') no-repeat;
+          background-color: $text-color2;
+        }
+        &.favorited .btn-icon {
+          mask: url('../img/icon-favorite-fill.svg') no-repeat;
```

Comment:
I got CORS errors trying to load this resource. Would you mind double-checking the NGINX config to see if any changes have to be made to allow loading SVGs from the CDN?